pkg/apis/config/defaults/image.go

```diff
@@ -18,4 +18,5 @@ limitations under the License.
 package defaults
 
 // Image is the default for the Config.Image field, aka the default node image.
-const Image = "kindest/node:v1.24.3@sha256:e1a5c449fb2556b5fbba29fb43f28055cacb12ce53d1f281e8be442d0dd5cd1d"
+// TODO: DO NOT MERGE with personal test image
+const Image = "docker.io/bentheelder/kind-node:v1.24.3@sha256:9fdef952a166325f89cae70b7e69767b50b7a7b0ace1f03df362228b82ee34e4"
```

Comment:
The provided image does not seem to contain the changes listed in this PR.
See https://github.com/kubernetes-sigs/kind/issues/2868#issuecomment-1211183768

Reply:
sorry, fixed!